### DISCUSSION TITLE
Inquiry Regarding Potential Hiring Scam Email - Verification Needed

### DISCUSSION THREAD

**Discussion by hungpixi** [community/community#109462] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Dear GitHub Support Team,
I hope this message finds you well. I am writing to seek clarification and verification regarding an email I received, which claims to be from GitHub. Given the nature of the content and the unsolicited approach, I am concerned that it might be a scam or phishing attempt, and I would greatly appreciate your assistance in confirming its legitimacy.
The email was sent from [notifications@github.com](mailto:notifications@github.com) and was received on 2:34AM (GMT+7) today. It mentions an "exciting opportunity" for a Developer position at GitHub, with an offer that includes a competitive salary and other benefits. The email also requests the completion of additional forms and provision of further information by clicking on a provided link, with a 24-hour deadline to complete the application process.
The project mentioned in the email is "Abadiq313/opensource.guide," and it references a pull request (PR #85) to bump actions/checkout from 2.3.5 to 4.1.1. I have concerns because the project and the context of the email do not align with my professional interests or activities on GitHub, leading me to suspect that this might be a phishing attempt or scam.
For your reference, here are some of the details included in the email:
Subject: Re: [Abadiq313/opensource.guide] Bump actions/checkout from 2.3.5 to 4.1.1 (PR #85)
Sent by: Github [notifications@github.com](mailto:notifications@github.com)
Date and Time: [insert date and time here]
Mentioned GitHub usernames: vadimsj, @felixdiria, @lamharrison, @TonGarcia, @lanzhiquan, @AbhimanyuAryan, @3lvis, @hackneycrypto, @akarju, @Tim9Liu9, @bnn9the, @tsaifeitingTaiwan, @taylorlee, @mtaarao, @liguopeng80, @Grizzly25, @jslauthor, @kchrs, @SiberSilanet, @yeomann
Could you please verify whether this email is genuinely from GitHub and advise on the appropriate steps I should take? Your prompt response would be greatly appreciated, as the email mentions a 24-hour deadline for the application process.
Thank you for your attention to this matter. I look forward to your swift response.
Best regards,
![image](https://github.com/community/community/assets/87779412/70e93c2a-ad9b-4665-b8f6-ba6648ee3d04)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @hungpixi ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

A version of this question has already been asked [here](https://github.com/orgs/community/discussions/109171) and you may find the comments there to be more useful.

**Reply by cococo2000** [reply]
It's quite peculiar that the discussion [https://github.com/orgs/community/discussions/109171](https://github.com/orgs/community/discussions/109171) is still ongoing in the afternoon, but now page not found 404???

**Comment by ettaboyle** [comment, marked as answer]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Comment by Gounlaf** [comment]
:wave: @ettaboyle 

> Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).

I did receive a notification like this.
I can't report anything because it's not on an issue, it's not a user, not a discussion, not a PR...

It's just a e-mail!

Is there an e-mail address where can I forward the full e-mail?

Regards.

**Reply by dennisvanderpool** [reply]
Same here, see attachment
[Re  [astrapi69_bundle-app-ui] Create panel for set password for root user (#1) - Microsoft Jobs (notifications@github.com) - 2024-03-04 1357.eml.zip](https://github.com/community/community/files/14482962/Re.astrapi69_bundle-app-ui.Create.panel.for.set.password.for.root.user.1.-.Microsoft.Jobs.notifications%40github.com.-.2024-03-04.1357.eml.zip)
